plugins/node.d/snmp__winmem

```diff
@@ -86,20 +100,20 @@ total.label Available memory
 total.info Total memory available
 total.draw LINE1
 EOC
+	print "used.warning ",  int($total * $ENV{warning}  / 100), "\n"
+	  if $ENV{warning};
+	print "used.critical ", int($total * $ENV{critical} / 100), "\n"
+	  if $ENV{critical};
```

Comment:
Here you could also use `print_thresholds("used");` (from `Munin::Plugin`), or?

See https://github.com/munin-monitoring/munin/blob/a2dec6c8afffbfa576c21c8d47036ba6ec22f5b5/lib/Munin/Plugin.pm#L373

Reply:
In this case you would probably also need `adjust_thresholds` for calculating absolute values based on percentage limits:

https://github.com/munin-monitoring/munin/blob/a2dec6c8afffbfa576c21c8d47036ba6ec22f5b5/lib/Munin/Plugin.pm#L389

Reply:
Thank you!
I guess, I can't use both `print_thresholds` and `adjust_threshold`, so I've chosen the latter.